proxies/live/apiproxy/resources/jsc/SetErrorVars.js

```diff
@@ -0,0 +1,210 @@
+// Regex to validate the id is a valid UUID
+const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi
+const nhsNumberRegex = /^[0-9]{10}$/gi
+const dobRegex = /^[0-9]{4}-[0-9]{2}-[0-9]{2}$/gi
+
+var data = context.getVariable("request.content.data")
+// var requiredParameters = [
+//   "type",
+//   "attributes/routingPlanId",
+//   "attributes/batchReference",
+//   "attributes/recipients",
+//   "attributes/recipients/0/recipientReference",
+//   "attributes/recipients/0/nhsNumber",
+//   "attributes/recipients/0/dateOfBirth",
+//   "attributes/recipients/0/personalisation",
+// ]
+// var requestParameters = context.getVariable("")
+var seenRecipients = []
```

Comment:
I thought it may be faster to do this lookup in a dict, but it isn't! The time to check if a value is within an array or within a dict is the same on node 16.